BD-1358: Update Message Steps doc GA (#3519)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/message_step.md

![The Delivery Settings tab for Message Step settings. Quiet Hours are enabled, and the checkbox for Using Intelligent Timing is selected to deliver the message at an optimal time.][4]

> For Canvas Message Steps, `event_properties` are not supported. Instead, use `canvas_entry_properties`. Canvas entry properties are properties from the event that kicked off the Canvas. These properties can only be used in the first full step of a Canvas. On the other hand, event properties originate from an event or action that occurs as the user goes through their workflow.

Reviewer: ```suggestion
For Canvas Message Steps, `event_properties` are not supported. Instead, use `canvas_entry_properties`. Canvas entry properties are properties from the event that started the Canvas. These properties can only be used in the first full step of a Canvas. Conversely, event properties originate from an event or action that occurs as the user goes through their workflow.
```
